src/main/g8/$name;format="normalize"$/src/main/scala/$package$/Application.scala

```diff
@@ -1,15 +1,51 @@
 package $package$
 
+import $package$.http._
 import $package$.db.repository.PersonRepository
 import $package$.utils.db.Migration
-import $package$.utils.log.Logable
-import zio._
-
-object Application extends Logable with ZIOAppDefault with PersonRepository {
-  def run =
-    for {
-      _      <- Migration.migrate
-      result <- getAllPersons
-      _      <- log.info(result.toString())
-    } yield ()
+import org.slf4j.LoggerFactory
+import sttp.tapir.server.interceptor.log.DefaultServerLog
+import sttp.tapir.server.ziohttp.{ZioHttpInterpreter, ZioHttpServerOptions}
+import zhttp.http.HttpApp
+import zhttp.service.server.ServerChannelFactory
+import zhttp.service.{EventLoopGroup, Server}
+import zio.{Console, Scope, Task, ZIO, ZIOAppArgs, ZIOAppDefault}
+
+object Application extends ZIOAppDefault with PersonRepository {
+
+  val log = LoggerFactory.getLogger(Application.getClass.getName)
+
+  override def run: ZIO[Any with ZIOAppArgs with Scope, Any, Any] = {
+    val serverOptions: ZioHttpServerOptions[Any] =
+      ZioHttpServerOptions.customiseInterceptors
+        .serverLog(
+          DefaultServerLog[Task](
+            doLogWhenReceived = msg => ZIO.succeed(log.debug(msg)),
+            doLogWhenHandled =
+              (msg, error) => ZIO.succeed(error.fold(log.debug(msg))(err => log.debug(msg, err))),
+            doLogAllDecodeFailures =
+              (msg, error) => ZIO.succeed(error.fold(log.debug(msg))(err => log.debug(msg, err))),
+            doLogExceptions = (msg: String, ex: Throwable) => ZIO.succeed(log.debug(msg, ex)),
+            noLog = ZIO.unit
+          )
+        )
+        .metricsInterceptor(PersonEndpoint.prometheusMetrics.metricsInterceptor())
+        .options
+
+    val app: HttpApp[Any, Throwable] =
+      ZioHttpInterpreter(serverOptions).toHttp(PersonEndpoint.all)
+
+    (for {
+      _ <- Migration.migrate
+      conf <- ZIO.service[HttpServerConfig]
+      port = conf.port
+      serverStart <- Server(app).withPort(port).make
+      _ <- Console.printLine(
+        s"Go to http://localhost:\${port}/docs to open SwaggerUI. Press ENTER key to exit."
+      )
+      _ <- Console.readLine
```

Comment:
Why do you think we need the server to go down with a single keystroke? Maybe remove it?